### PR TITLE
docs: 📝 add section on merging topic branches

### PR DESCRIPTION
**Pull request by signekb**
## Description

I have added this section based on talks me and @lwjohnst86 has had about how to merge topic branches. We're trying to create a set of simple rules to follow when merging branches into `main`.

## Reviewer Focus

This PR needs an in-depth review.

## Checklist

- [X] Ran spell-check
- [X] Formatted Markdown
- [X] Rendered website locally

### CODE REVIEW

**Comment by signekb** [comment]
Yes, @K-Beicher! I just formatted all text in the post which resulted in Git highlighting changes throughout

File: how-we-work/admin.qmd [hunk 1]
```diff
@@ -3,45 +3,89 @@ title: "General admin content"
...
 gh repo create seedcase-project/REPO --public --source=. --description="WRITE DESCRIPTION" --push --disable-wiki --homepage https://REPO.seedcase-project.org 
 ```
+
+## Merging topic branches
+
+Since we use
+[SemVer](https://decisions.seedcase-project.org/why-semver/) and want to
+have a more automated release process (including automated changelogs),
+we need to have a clear and consistent way of merging topic branches.
+This way, we ensure that the changes are made in each release can be
```
Comment (martonvago) [inline how-we-work/admin.qmd:74]:
```suggestion
This way, we ensure that the changes made in each release can be
```

File: how-we-work/admin.qmd [hunk 2]
```diff
@@ -3,45 +3,89 @@ title: "General admin content"
...
+decide which kind of merge to perform on topic branches, we follow these
+general guidelines:
+
+-   **Squash:** Atomic PRs
+
+-   **Rebase:** Build, chore, and sync
+
+-   **Merge:** Non-atomic/other PRs
+
+With atomic PRs, we mean PRs that only include "self-contained changes"
```
Comment (martonvago) [inline how-we-work/admin.qmd:88]:
```suggestion
By atomic PRs, we mean PRs that only include "self-contained changes"
```

**Review by martonvago — CHANGES_REQUESTED** [review]
👍 👍 👍

File: how-we-work/admin.qmd [hunk 3]
```diff
@@ -3,45 +3,89 @@ title: "General admin content"
...
+easily traced.
+
+In our repositories, we follow the [GitHub
+flow](https://decisions.seedcase-project.org/why-github-flow/), and to
+decide which kind of merge to perform on topic branches, we follow these
+general guidelines:
+
+-   **Squash:** Atomic PRs
+
+-   **Rebase:** Build, chore, and sync
```
Comment (lwjohnst86) [inline how-we-work/admin.qmd:84]:
```suggestion
-   **Rebase:** Commits or PRs that have `build` and `chore` types as well as `sync` scopes (which usually fall under `chore` type)
```

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]
Nice! Very small edit.

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**